packages/main/src/Badge.ts

```diff
@@ -51,8 +51,8 @@ import badgeCss from "./generated/themes/Badge.css.js";
 class Badge extends UI5Element {
 	/**
 	 * Defines the color scheme of the component.
-	 * There are 10 predefined schemes. Each scheme applies different values for the <code>background-color</code> and <code>border-color</code>.
-	 * To use one you can set a number from <code>"1"</code> to <code>"10"</code>. The <code>colorScheme</code> <code>"1"</code> will be set by default.
+	 * There are 10 predefined schemes.
+	 * To use one you can set a number from <code>"1"</code> to <code>"10"</code>. The default <code>colorScheme</code> will be dependent on the loaded theme.
```

Comment:
Did not get this part, <code>"1"</code> should be set by default right?

Reply:
Well yes. But in the Horizon specification, the default color scheme (meaning the attribute `color-scheme="..."` not existing on the element) is defined to be the `color-scheme="7"`. Accessing the property with the default value will return you "1", but in Horizon the default colors will really be from "7".

Reply:
For me it really strange to have different behaviour when following to lines are used:
```html
<ui5-badge>Test</ui5-badge>
<ui5-badge color-scheme="1">Test</ui5-badge>
```

It seems that the default value of this API is changed to undefined which is breaking change.

Reply:
What I can do is ignore the text in the specification that says the default color scheme in Horizon should be "7". What do you think?

Reply:
The default value for all themes should be 7
This is the initial design (see openui5 - https://sapui5untested.int.sap.eu2.hana.ondemand.com/#/api/sap.tnt.InfoLabel%23controlProperties)
The problem with WebC Badge is that it is set to 1, which is wrong.
The default value should be changed to 7 for all themes. Is it possible?

Reply:
Hi @gmkv, @LidiyaGeorgieva,

This wasn't mentioned in the specification when the component was introduced. Changing the default value will be a breaking change and could be considered for 2.0 change. This change is added in 2.0 breaking change list (https://github.com/SAP/ui5-webcomponents/issues/4460)